src/phoenix/db/migrations/versions/10460e46d750_datasets.py

```diff
@@ -216,6 +216,15 @@ def upgrade() -> None:
             sa.String,
             nullable=False,
         ),
+        sa.Column(
+            "annotator_kind",
+            sa.String,
+            sa.CheckConstraint(
+                "annotator_kind IN ('LLM', 'HUMAN')",
```

Comment:
```suggestion
                "annotator_kind IN ('LLM', 'CODE', 'HUMAN')",
```
We are going to have pure code based evaluations so we should probably leave room for that

Reply:
addressed in de0df5f60ab4b38672f2e289c599b1f24e8851da